CT-1552 Update business units with SARS responding rights

stephenrichards opened the pull request.

misaka commented on app/policies/case_policy.rb:
```diff
@@ -23,6 +23,10 @@ def can_add_case?
     user.manager?
   end
 
+  def can_approve_case?
```
I don't think we need this section, although I do like this name better than `can_approve?`.

stephenrichards committed:
CT-1552 Update Business Units requirings SARS responding rights

This introduces two new rake tasks:

 * rake kilo_map:load_sars[<filename>] Reads the CSV file <filename> and updates
   those business units marked as requiring SARS rights.  The file is currently
   in lib/assets

 * rake kilo_map:audit_corr_types Prints a list of business units and what
   correspondence types they are authorised to respond to

After deployment to production, the following command must be run manually:

   rake kilo_map:load_sars[lib/assets/kilo_map.csv]

stephenrichards committed:
Merge branch 'master' into ct-1552-update-kilo-map-with-sars
